index.html

```diff
@@ -29,7 +29,7 @@
           </tr>
           <tr>
             <td>Remaining Calories</td>
-            <td class="remaining-calories"></td>
+            <td></td>
```

Comment:
Why did you remove the `remaining-calories` class. It would make this field easier to find via jQuery. I see you using the selector `tfoot tr:nth-child(2) td:nth-child(2)` later on, but that could be just `.remaining-calories`.

Was there a reason you took this out?